cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
I agree the previous messaging is cumbersome.  Printing just a message like `Initializing FS backend` or 'Initializing backend format` is good enough.   Complex code for this would cause penalty in code maintainability.